Gruntfile.js

```diff
@@ -1,20 +1,28 @@
 'use strict';
 module.exports = function (grunt) {
+
+    // Set the video.js version
+    var vjsVersion = '5.2.1';
```

Comment:
this should probably be passed in as an argument to grunt or grab it from the package.json of videojs.